.github/workflows/msvc-build.yml

```
@@ -70,12 +78,19 @@ jobs:
 
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
       
+      - name: Setup Visual Studio 2015 Build Tools
+        run: |
+          # 下载并安装 Visual Studio 2015 Build Tools
+          Invoke-WebRequest -Uri "https://go.microsoft.com/fwlink/?LinkId=691126" -OutFile "visualcppbuildtools_full.exe"
+          Start-Process -FilePath "visualcppbuildtools_full.exe" -ArgumentList "/S" -NoNewWindow -Wait
+        shell: powershell
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)